src/user/user.controller.ts

```diff
@@ -23,10 +23,4 @@ export class UserController {
     this.logger.log('GET /user/me called');
     return user;
   }
-
-  @Get()
-  getHello(): string {
-    this.logger.log('GET /user called');
-    return 'Hello World!';
-  }
```

Comment:
@coderabbitai Here are the lines that were removed in commit fd718a1 (full commit id: `fd718a128e30668711bbb3de24d47cef42c8571b`).